plugins/rbac/src/components/RolesList.tsx

```diff
@@ -61,6 +66,17 @@ export const RolesList = () => {
           {toastMessage}
         </Alert>
       </Snackbar>
+      <span className={classes.createRoleButton}>
```

Comment:
Frustrating that the backstage components seem to avoid having a `className` property.  Anyway, it's really only a nit but I suggest maybe call this class `createRoleButtonWrapper` just so it's more clear earlier in the file where this classname is defined that it doesn't target the button but instead a wrapper component that's used to align the button.